extensions/ql-vscode/src/databases/db-manager.ts

```diff
@@ -194,24 +191,44 @@ export class DbManager {
     );
   }
 
-  private async updateExpandedItems(items: ExpandedDbItem[]): Promise<void> {
+  private async updateExpandedItems(
+    newExpandedItems: ExpandedDbItem[],
+  ): Promise<void> {
     let itemsToStore;
 
-    const dbItemsResult = this.getDbItems();
+    const dbItems = this.getDbItems();
```

Comment:
When you do `.getDbItems()` you don't get the dbItems, you get a result object. I wouldn't rename this personally (but up to you).